modules/cert-manager-configure-cloud-credentials-aws-sts.adoc

```diff
@@ -0,0 +1,133 @@
+// Module included in the following assemblies:
+//
+// * security/cert_manager_operator/cert-manager-authenticate-aws.adoc
+
+:_content-type: PROCEDURE
+[id="cert-manager-configure-cloud-credentials-aws-sts"]
+= Configuring cloud credentials for the {cert-manager-operator} for the AWS Security Token Service cluster
+
+To configure the cloud credentials for the {cert-manager-operator} on the AWS Security Token Service (STS) cluster you must use the `ccoctl` binary to generate the cloud credentials manually and apply it on the cluster.
+
+.Prerequisites
+
+* You must extract and prepare the `ccoctl` binary.
+* The {cert-manager-operator} 1.11.1 or later is installed.
```

Comment:
As mentioned here https://github.com/openshift/openshift-docs/pull/59717#issuecomment-1571773933 yesterday, in AWS STS there are 2 approaches, one using `eks.amazonaws.com/role-arn` annotation which worked already in 1.10.2, so for this approach using `eks.amazonaws.com/role-arn` annotation, need not the per-requisite `The {cert-manager-operator} 1.11.1 or later is installed.`